### ISSUE TITLE
Issue with mj-html-attributes and templating language tags <% %>

### ISSUE THREAD

**Issue by cantoniriccardo** [mjmlio/mjml#2744]
**Describe the bug**
When we use mj-html-attributes and a templating language like EJS inside a compontent, a tag <%= my_variable %> is not preserved as is, but it is rendered as <%= my_variable="" %="" />.

**To Reproduce**
1. Copy and paste the following code in the try-it-live tool: 
```
<mjml>
  <mj-head>
    <mj-html-attributes>
      <mj-selector path=".custom div">
        <mj-html-attribute name="data-id">42</mj-html-attribute>
      </mj-selector>
    </mj-html-attributes>
  </mj-head>
  <mj-body>
    <mj-section>
      <mj-column>
        <mj-text css-class="custom">
          Hello <%= world %>
        </mj-text>
      </mj-column>
    </mj-section>
  </mj-body>
</mjml>
```
3. See error

**Expected behavior**
We expect the tag <% ... %> to be rendered as is.

**MJML environment**
 - MJML Version 4.9.3
 - MJML try-it-live tool

**Comment by iRyusa** [comment]
Yup, I think this should be mentioned in the doc as it's not something we can fix. When Cheerio parse the document, any "look alike" xml/html will be treated as a xml tag so it will alter the output HTML
